src/code/z_parameter.c

```diff
@@ -3874,11 +3874,11 @@ void Interface_DrawItemButtons(PlayState* play) {
                                            D_801BF9E4[1] * 2, D_801BF9E4[1] * 2, 255, 240, 0, interfaceCtx->cLeftAlpha);
     // C-Down Button Color & Texture
     OVERLAY_DISP = Gfx_DrawRect_DropShadow(OVERLAY_DISP, D_801BF9D8[0], D_801BF9E0[0], D_801BFAF8[0], D_801BFAF8[0],
-                                           D_801BF9E4[2] * 2, D_801BF9E4[2] * 2, 255, 240, 0, interfaceCtx->cDownAlpha);
+                                           D_801BF9E8[0] * 2, D_801BF9E8[0] * 2, 255, 240, 0, interfaceCtx->cDownAlpha);
```

Comment:
I think it makes more sense to keep it as `D_801BF9E4` and to just merge `D_801BF9E4` and `D_801BF9E8` into a single piece of data:
`s16 D_801BF9E4[] = { 0x23F, 0x26C, 0x26C, 0x26C};`
then you can have 
`D_801BF9E4[EQUIP_SLOT_B]`
`D_801BF9E4[EQUIP_SLOT_C_LEFT]`
`D_801BF9E4[EQUIP_SLOT_C_DOWN]`
`D_801BF9E4[EQUIP_SLOT_C_RIGHT]`

Same with `D_801BF9DC`/`D_801BF9E0` and `D_801BF9D4`/`D_801BF9D8` (that's also UB right now in the `// Empty C Button Arrows` loop)

Reply:
Nice catch! I also added a few comments with the indices in each array